README.md

```diff
@@ -7,15 +7,28 @@
+    
+[KNE](https://github.com/google/kne) automates this process and enables us to setup network topologies in Kubernetes. It uses [Meshnet](https://github.com/networkop/meshnet-cni) CNI to setup the network interconnects. Ixia Operator watches out for IxiaTG CRDs to be instantiated in Kubernetes environment and accordingly initiates Ixia specific resource management.<br/>
+    
+The various Ixia component versions to be deployed is derived from the Ixia release version as specified in the IxiaTG config. These component mappings are captured in ixia-configmap.yaml for each Ixia release. Ixia Operator first tries to access these details from Keysight published releases; in case of failure or in absence of internet connectivity, it tries to locate them in Kubernetes configmap. Thus, for the second scenario, the user is expected to separately download release specific ixia-configmap.yaml from published [releases](https://github.com/open-traffic-generator/ixia-c/releases/), and apply it locally; after the operator is started.<br/>
 
-Ixia Operator defines CRD for Ixia network device (IxiaTG) and can be used to build up different network topologies possibly with network devices from other vendors. Network interconnects between the topology nodes can be setup with various container network interface (CNI) plugins for Kubernetes for attaching multiple network interfaces to the nodes.
+Below is a snippet of a ixia-configmap.yaml. Users can deploy custom versions by applying a configmap with custom updates and then specifying that custom version in KNE config file.<br/><br/>
 
-[KNE](https://github.com/google/kne) automates this process and enables us to setup network topologies in Kubernetes. It uses [Meshnet](https://github.com/networkop/meshnet-cni) CNI to setup the network interconnects. Ixia Operator watches out for IxiaTG CRDs to be instantiated in Kubernetes environment and accordingly initiates Ixia specific resource management. Based on the node version in the IxiaTG config, it first tries to access the deployment resource mapping details from Keysight published [releases](https://github.com/open-traffic-generator/ixia-c/releases/); in case of failure or in absence of internet connectivity, it refers to local configmap. This way it allows user to deploy custom versions by allowing load of a configmap with custom updates and then specifying that custom version in KNE config file. If some of the required images are not publicly available and distributed under license, user would be required to create a secret with docker authorization details as supplied by Keysight. The operator would refer to this secret for deploying from private repository.
+<kbd> <img src="ixia-configmap.jpg" title="Sample configmap"> </kbd><br/><br/>
```

Comment:
@anjan-keysight : Can we include a text file instead of an image .  And if possible, add some possible tags for custom "command", "args" or "env" where relevant ? So easier for someone to copy/paste/update as needed

Reply:
@apratimmukherjee Will add that.